tests/e2e/dsc_creation_test.go

```diff
@@ -282,7 +284,7 @@ func (tc *testContext) testOwnerrefrences() error {
 	// Test any one of the apps
 	if tc.testDsc.Spec.Components.Dashboard.ManagementState == operatorv1.Managed {
 		appDeployments, err := tc.kubeClient.AppsV1().Deployments(tc.applicationsNamespace).List(context.TODO(), metav1.ListOptions{
-			LabelSelector: "app.opendatahub.io/" + tc.testDsc.Spec.Components.Dashboard.GetComponentName(),
+			LabelSelector: odhLabel + tc.testDsc.Spec.Components.Dashboard.GetComponentName(),
```

Comment:
I think we could call it `odhLabelPrefix`

Reply:
done